CloudFormation/arkcase.yml

```diff
@@ -607,6 +609,79 @@ Resources:
         - Key: ManagedBy
           Value: CloudFormation
 
+  ###########################################################
+  # Task definition custom resource with EFS specifications #
+  ###########################################################
+
+  TaskDefinitionResourceLambdaExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: sts:AssumeRole
+      Policies:
+        - PolicyName: TaskDefinitionResourceLambaPolicy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Action:
+                  - ecs:RegisterTaskDefinition
+                  - ecs:DeregisterTaskDefinition
+                Resource: "*"
+                Effect: Allow
+
+              - Action:
+                  - iam:GetRole
+                  - iam:PassRole
+                Resource: "*"
+                Effect: Allow
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+      Tags:
+        - Key: Name
+          Value: !Sub task-definition-resource-lambda-execution-role-${Project}-${Env}
+        - Key: Env
+          Value: !Ref Env
+        - Key: Project
+          Value: !Ref Project
+        - Key: Service
+          Value: global
+        - Key: Component
+          Value: iam
+        - Key: ManagedBy
+          Value: CloudFormation
+
+  TaskDefinitionResourceLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Description: >
+        CloudFormation custom resource backend to create a task definition that
+        also supports EFS mounts
+      Runtime: python3.6
+      Role: !GetAtt TaskDefinitionResourceLambdaExecutionRole.Arn
+      Handler: task_definition_resource.handler
+      Timeout: 30
+      Code:
+        S3Bucket: !Sub arkcase-public-${AWS::Region}
+        S3Key: DevOps/ACM-TMP-20201210-0907/LambdaFunctions/task_definition_resource/task_definition_resource.zip
```

Comment:
Is the source code and packaging script for this lambda also in this same repository, or is it somewhere else?